app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -15,7 +16,7 @@ def self.binding_address
 
     def self.rails_server_command_line
       @rails_server_command_line ||= begin
-        "#{self.nice_prefix} #{Rails.root.join("bin", "rails")} server".freeze
+        "#{nice_prefix} #{Rails.root.join("bin", "rails")} server #{RAILS_SERVER}".freeze
```

Comment:
@matthewd 
Are you asking why not in-line `server puma` vs. using a constant?

My thought is we might want to access `MiqUiWorker::RAILS_SERVER` and we've had `thin` buried in method names and method bodies (way before this PR) so we had multiple places to find/change what rails server we used.